python/ml4ir/base/config/parse_args.py

```diff
@@ -331,6 +331,27 @@ def define_args(self):
             help="[Ranklib format only] Option to keep additional info (All info after the '#' in the format [key = val]). 1 to keep, 0 to ignore",
         )
 
+        self.add_argument(
+            "--write_tf_model_dir",
```

Comment:
@balikasg Can we maybe add "bundle" or "rich_model_bundle" something like that to differentiate?